### DISCUSSION TITLE
Bug: Code tags within links wrap last-letter on Firefox

### DISCUSSION THREAD

**Discussion by AnilSeervi** [gatsbyjs/gatsby#34377] [Help] [answered]
## Problem as seen on Firefox
![image](https://user-images.githubusercontent.com/61609033/147843311-be98a289-947c-4017-96c5-ed3fdd61442c.png)

`<code>` blocks inside the `.main-body a` tags wrap the last-letter as shown in picture above.
Page-link - https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v1-to-v2/

Affected pages: All pages on `/docs`

**Comment by AnilSeervi** [comment, marked as answer]
## Motivation
`<a>` tags inside the `.main-body` class have the property `word-break: break-word;` this property allows the text to break at words.
Note: its a **deprecated** value

### The cause of Problem on Firefox
So I digged into the devTools and after like a hour of inspection found that _Firefox_ is actually inheriting the `work-break` property to the `::before, ::after` elements of the `<code`> blocks, causing the last letter of text inside `<code>` to `word-break`.

### Solution/Suggestion
As we know what's causing the problem here, the optimal solution would be to add `work-break: normal;` to the `::before, ::after` elements of the `<code`> block.
```css
code::before, code::after, tt::before, tt::after {
    letter-spacing: -0.2em;
    content: " ";
    word-break: normal;
}
```

**Reply by fk** [reply]
Thank you @AnilSeervi for opening an issue and providing a detailed analysis and fix here – super appreciated! 🙏 <3
Sorry for taking long – a fix for this is about to be deployed.

**Reply by AnilSeervi** [reply]
Thanks for fixing the issue. I just confirmed that the fix has been deployed and works well on Firefox while keeping the same format on other browsers too.
